src/datasets/iterable_dataset.py

```diff
@@ -929,6 +961,10 @@ def iter_arrow(self):
         if self.formatting and self.formatting.format_type == "arrow":
             return self._iter_arrow
 
+    @property
+    def is_typed(self):
+        return self.ex_iterable.is_typed
```

Comment:
we can't know the output of a mapped iterable, so it should be False no ?
```suggestion
        return False
```

Reply:
oh yes - should be False

still, wondering what happens if user passes explicit features != None to map:
in that case should the feature conversion be applied or not? If it should be applied I think returning False here works, if it shouldn't we may need to handle that somehow

Reply:
If we pass features != None to map(), the ExampleIterable is not typed so the IterableDataset will apply the feature conversion in `IterableDataset.__iter__`. And if there is a subsequent map(), it will use a TypedExampleIterable as input to the subsequent map function so the feature conversion is also applied.